http-client/Network/HTTP/Client/Connection.hs

```diff
@@ -11,6 +11,7 @@ module Network.HTTP.Client.Connection
     , makeConnection
     , socketConnection
     , withSocket
+    , strippedHostName
```

Comment:
Not really sure where this belongs in the module hierarchy.

Reply:
This seems fine to me.